docs/development/contribution-documentation/documentation-process-internal-contributor/README.md

```diff
@@ -33,14 +33,14 @@ In the modal window select the repository ""*opf/openproject*"". Also select a f
 
 > **Important**: Before you make changes always update your local repository.
 
-1. Select the branch you want to work on, e.g.`release/12.1` in the main toolbar.
+1. Select the branch you want to work on, e.g. `release/12.1` in the main toolbar.
```

Comment:
@cbliard sorry, forget my last comment it is a different file you are editing, its about internal contributors
could you maybe fix 12.1 to 12.2 in the whole document? not the screenshots but the text, thank you

Reply:
In fact, I have no idea which files I was editing or the content. I saw some spacing issues in one file. I fixed them, then I did a search to find similar issues and fixed them too. I also fixed some minor typo along the way.

I'll change the text 12.1 to 12.2 where relevant as requested.

Reply:
Well, in fact I'll keep it like this as the screenshots of this document are all made with `release/12.1` on it. Changing it to release `release/12.2` would make it inconsistent, or require new screenshots, which is not really needed to understand the point of this doc.

Is it ok for you @adam-op?

Reply:
I updated it as requested.

Reply:
The screenshots are not necessary to be updated, maybe only for major versions. Thank you for your help